backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/export/svg/EdgeExportService.java

```diff
@@ -142,9 +142,9 @@ private StringBuilder exportArrow(Position p1, Position p2, String arrowStyle, E
         } else if (styleName.equals("InputFillClosedArrow")) { //$NON-NLS-1$
             arrowExport = this.exportInputClosedArrow(p1, p2, style, true);
         } else if (styleName.equals("Diamond")) { //$NON-NLS-1$
-            arrowExport = this.exportDiamondArrow(p1, p2, style, false);
+            arrowExport = this.exportDiamondArrow(p2, p1, style, false);
```

Comment:
If the method `exportDiamondArrow` has a bug, it would be better to fix it there instead of everywhere this method is called.

Reply:
In the "Diamond" and "FillDiamond" styles the arrow is on the source-side of the edge instead of the target-side, so we want to switch sides when positionning the arrow, I'm not sure it can be considered an error of the exportDiamondArrow method so to speak